CIP-0030/README.md

```diff
@@ -54,7 +54,7 @@ A string representing an address in either bech32 format, or hex-encoded bytes.
 
 A hex-encoded string of the corresponding bytes.
 
-#### cbor\<T>
+#### `cbor<T>`
```

Comment:
```suggestion
#### cbor<T>

```ts
type cbor<T> = string;
```
What about something like this? more consistent with the other datatype section titles?

Reply:
I wouldn't mind adding some code afterward to clarify the data type (though I already think it's clear from the paragraph text that follows)... but the `cbor<T>` needs to be quoted; otherwise it just appears as `cbor`, with Showdown dropping the angle bracketed text, as you can see at this section header: https://developers.cardano.org/docs/governance/cardano-improvement-proposals/cip-0030/#cbor

Reply:
Ahh I see, lets keep it simple then, what youve suggested with `cbor<T>` makes sense.